tests/queries/0_stateless/02188_table_function_file.sql

```diff
@@ -0,0 +1,68 @@
+select * from format(JSONEachRow, 
```

Comment:
What's for `02188_table_function_file.sql` ? Is it should be only `02188_table_function_format.sql` ?

Reply:
It should be only `02188_table_function_format.sql`, just forgot to delete old file after bad renaming